catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_002.jsonnet

```diff
@@ -0,0 +1,3431 @@
+      center_wavelength: 2.4458398819,
+      full_width_half_max: 0.0063717905,
+      'gee:wavelength': '2445.8374nm, FWHM: 6.3718nm',
+     },
+     {
+      name: "B414",
+      description: "SWIR Band B414",
+      center_wavelength: 2.4508500099,
+      full_width_half_max: 0.0063761217,
+      'gee:wavelength': '2450.846nm, FWHM: 6.3761nm',
+     },
+     {
+      name: "B415",
+      description: "SWIR Band B415",
+      center_wavelength: 2.4558498859,
+      full_width_half_max: 0.0063804626,
+      'gee:wavelength': '2455.855nm, FWHM: 6.3805nm',
+     },
+     {
+      name: "B416",
+      description: "SWIR Band B416",
+      center_wavelength: 2.460860014,
+      full_width_half_max: 0.0063848165,
+      'gee:wavelength': '2460.8635nm, FWHM: 6.3848nm',
+     },
+     {
+      name: "B417",
+      description: "SWIR Band B417",
+      center_wavelength: 2.4658699036,
+      full_width_half_max: 0.0063891839,
+      'gee:wavelength': '2465.8723nm, FWHM: 6.3892nm',
+     },
+     {
+      name: "B418",
+      description: "SWIR Band B418",
+      center_wavelength: 2.4708800316,
+      full_width_half_max: 0.0063935672,
+      'gee:wavelength': '2470.881nm, FWHM: 6.3936nm',
+     },
+     {
+      name: "B419",
+      description: "SWIR Band B419",
+      center_wavelength: 2.4758899212,
+      full_width_half_max: 0.0063979598,
+      'gee:wavelength': '2475.89nm, FWHM: 6.398nm',
+     },
+     {
+      name: "B420",
+      description: "SWIR Band B420",
+      center_wavelength: 2.4809000492,
+      full_width_half_max: 0.0064023668,
+      'gee:wavelength': '2480.8987nm, FWHM: 6.4024nm',
+     },
+     {
+      name: "B421",
+      description: "SWIR Band B421",
+      center_wavelength: 2.4859099388,
+      full_width_half_max: 0.0064067845,
+      'gee:wavelength': '2485.907nm, FWHM: 6.4068nm',
+     },
+     {
+      name: "B422",
+      description: "SWIR Band B422",
+      center_wavelength: 2.4909200668,
+      full_width_half_max: 0.0064112167,
+      'gee:wavelength': '2490.9158nm, FWHM: 6.4112nm',
+     },
+     {
+      name: "B423",
+      description: "SWIR Band B423",
+      center_wavelength: 2.4959199429,
+      full_width_half_max: 0.0064156638,
+      'gee:wavelength': '2495.9243nm, FWHM: 6.4157nm',
+     },
+     {
+      name: "B424",
+      description: "SWIR Band B424",
+      center_wavelength: 2.5009300709,
+      full_width_half_max: 0.0064201197,
+      'gee:wavelength': '2500.933nm, FWHM: 6.4201nm',
+     },
+     {
+      name: "B425",
+      description: "SWIR Band B425",
+      center_wavelength: 2.5059399605,
+      full_width_half_max: 0.0064245905,
+      'gee:wavelength': '2505.9417nm, FWHM: 6.4246nm',
+     },
+     {
+      name: "B426",
+      description: "SWIR Band B426",
+      center_wavelength: 2.5109500885,
+      full_width_half_max: 0.0064290729,
+      'gee:wavelength': '2510.9507nm, FWHM: 6.4291nm',
+     },
+     {
+      name: 'B427',
+      'gee:units':'deg',
```

Comment:
units.degree here

Reply:
checking that the latest version is up. I updated all these units to match what's in the 001 catalog

Reply:
see comment above & https://github.com/google/earthengine-catalog/pull/1058/commits/b94ae22110e294378dd771521a6105e68c823d9e

Reply:
resolved in https://github.com/google/earthengine-catalog/pull/1058/commits/243970724367580ce1b059d3276e8315342a5f93